### PR TITLE
Triangle

### PR DESCRIPTION
**Pull request by dviduthra**
Added code to find factorial of n and n-th Fibonacci number

### CODE REVIEW

**Review by OarpitO — COMMENTED** [review]
@dviduthra Can you please write test cases for both of these programs? And check the results against your code?

Also, make sure you add description to the PR when you create one? For eg: What does the PR include? Is it a new program/change/bug fix on the existing code? Also the Test cases and results against those once you verified.

**Review by OarpitO — APPROVED** [review]

File: Fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    elif n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    elif n < 0:
+        return "Enter the value above 0"
+    else:
+        return fibonacci(n - 1) + fibonacci(n - 2)
+
+print(fibonacci(8))
```
Comment (OarpitO) [inline Fibonacci.py:14]: You can take user input over here.

File: factorial.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    if type(n) is not int:
+        return ("Enter the value in numbers")
+    elif n < 0:
+        return  ("Enter the value greater than or equal to 0")
+    elif n == 0:
+        return 1
+    else:
+        return n * factorial(n - 1)
+
+print(factorial())
```
Comment (OarpitO) [inline factorial.py:11]: You can take user input over here.

File: Fibonacci.py (hunk 1 above)
Comment (dviduthra) [reply Fibonacci.py]: Added,  Please review

File: factorial.py (hunk 2 above)
Comment (dviduthra) [reply factorial.py]: Added, please review

**Review by OarpitO — APPROVED** [review]

**Pull request merged**